pkg/router/rpc_rule.go

```diff
@@ -75,7 +76,7 @@ func (srri *RPCRouteRuleImpl) Match(ctx context.Context, headers api.HeaderMap)
 		}
 	}
 	if log.Proxy.GetLogLevel() >= log.DEBUG {
-		log.Proxy.Debugf(ctx, RouterLogFormat, "sofa rotue rule", "failed match, macther %s", srri.fastmatch)
+		log.Proxy.Debugf(ctx, RouterLogFormat, "Match", "sofa rotue rule", fmt.Sprintf("failed match, macther %s", srri.fastmatch))
```

Comment:
Nice catch!
`failed match` => `failed to match`, otherwise, it looks good to me.

Reply:
ok